### DISCUSSION TITLE
Is there any way to serve Next JS app and static website (as landing page) under one domain?

### DISCUSSION THREAD

**Discussion by jelen91** [vercel/next.js#42150] [Help]
Hi,
I have been following this article

https://vercel.com/guides/how-can-i-serve-multiple-projects-under-a-single-domain

The problem is I have next JS app and static websites (only html file and couple of css)
Is there anything I can add to the config file to make these 2 projects work together?

**Comment by Chrysippus** [comment]
Can you put the static files for the landing page in the 'public' folder? You'd just have to name them appropriately, I guess.
